src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write code here
+const MAX_SIZE = 10;
+const MIN_SIZE = 2;
+
+function refreshButtons(size, appendButton, removeButton) {
+  if (size >= MAX_SIZE) {
+    appendButton.disabled = true;
+  } else if (size <= MIN_SIZE) {
+    removeButton.disabled = true;
+  } else {
+    appendButton.disabled = false;
+    removeButton.disabled = false;
+  }
+}
+
+function resizeTable(tableEvent) {
+  const buttonClass = tableEvent.target.className.replace('button', '').trim();
+
+  switch (buttonClass) {
+    case 'append-row':
+      tBody.append(tBody.lastElementChild.cloneNode(true));
+      break;
+
+    case 'remove-row':
+      tBody.lastElementChild.remove();
+      break;
+
+    case 'append-column':
+      for (const row of tBody.children) {
+        row.append(row.lastElementChild.cloneNode());
+      }
+      break;
+
+    case 'remove-column':
+      for (const row of tBody.children) {
+        row.lastElementChild.remove();
+      }
+  }
```

Comment:
Can we create a function like button action and move the switch case into this function?

and the use it here? It will improve readability and make resizeable shorter